test/plugins.js

```diff
@@ -106,7 +106,7 @@ test.serial('should instantiate plugins and execute all release-cycle methods',
     t.is(myLocalPlugin[method].callCount, 1);
   });
 
-  const incrementBase = { latestVersion: '0.0.0', increment: 'patch', isPreRelease: false, preReleaseId: undefined };
+  const incrementBase = { latestVersion: '0.0.0', increment: undefined, isPreRelease: false, preReleaseId: undefined };
```

Comment:
@webpro Due to my change the options.increment is no longer modified via getIncrementedVersionCI. I guess that this is not a problem. But better check carefully if this change has side effects.